### PR TITLE
invalid relationship value Exception

### PR DESCRIPTION
**Pull request by fogelito**
Add a new type of exception specific for the relationship values .

## What does this PR do?

(Provide a description of what this PR does and why it's needed.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/controllers/general.php [hunk 1]
```diff
@@ -847,6 +847,10 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
                 \var_dump('Wrapping authorization exception');
                 $error = new AppwriteException(AppwriteException::USER_UNAUTHORIZED);
                 break;
+            case 'Utopia\Database\Exception\InvalidRelationshipValue':
+                \var_dump('Wrapping relationship exception');
+                $error = new AppwriteException(AppwriteException::RELATIONSHIP_VALUE_INVALID);
```
Comment (abnegate) [inline app/controllers/general.php:852]: Let's pass the message from the original exception into the appwrite exception too

File: app/controllers/general.php [hunk 2]
```diff
@@ -847,6 +847,10 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
                 \var_dump('Wrapping authorization exception');
                 $error = new AppwriteException(AppwriteException::USER_UNAUTHORIZED);
                 break;
+            case 'Utopia\Database\Exception\InvalidRelationshipValue':
+                \var_dump('Wrapping relationship exception');
```
[the remark this answers is not in the document]
Comment (fogelito) [reply app/controllers/general.php]: Done!

File: app/controllers/general.php (hunk 1 above)
Comment (fogelito) [reply app/controllers/general.php:844]: Done!

**Review by abnegate — APPROVED** [review]
Looks good, let's get the db side sorted then we can update here :+1:

**Comment by abnegate** [comment]
Pulled into https://github.com/appwrite/appwrite/pull/8036

**Pull request closed without merging**